responses/es/10_more-to-learn.md

```diff
@@ -0,0 +1,212 @@
+## Buen trabajo
+
+Enhorabuena @{{ user.username }}, ¡has completado este curso!
+
+![enhorabuena](https://octodex.github.com/images/welcometocat.png)
+
+## Lo que ha ido bien
+
+En este curso has conseguido:
+
+- Añadir una lista de tareas a un comentario en una propuesta
+- Activar GitHub Pages en tu repositorio
+- Confirmado cambios a tu sitio web, en concreto:
+    - Añadiste encabezados al archivo `01-name.md`
+    - Insertaste una imagen en el archivo `02-image.md` 
+    - Creaste enlaces a otros sitios web en el archivo `03-links.md` 
+    - Añadiste una lista de tus logros en el arhivo `04-lists.md` 
+    - Usaste énfasis como **negrita** y _cursiva_ en el archivo `05-emphasis.md` 
+
+## ¿Y ahora, qué?
+
+Ahora que ya controlas lo más básico de Markdown, @{{ user.username }}, aquí tienes algunas informaciones rápidas acerca de otras funcionalidades interseantes que no hemos visto.
+
+### Más enlaces en Markdown
+
+<details>
+<summary>@menciones de Usuarios y Equipos</summary>
+
+  ### @menciones de Usuarios y Equipos
+
+  Si tecleas el símbolo `@` seguido de un nombre de usuario de GitHub, se enviará una notificación a esa persona acerca de tu comentario. Esto se llama una "@mencion", porque estás mencionando a otra persona. También puedes `@mencionar` equipos dentro de una misma organización. Para más información, puedes consultar “[Receiving notifications about activity on GitHub](https://help.github.com/categories/receiving-notifications-about-activity-on-github/) en *GitHub Help*.
```

Comment:
```suggestion
  Si tecleas el símbolo `@` seguido de un nombre de usuario de GitHub, se enviará una notificación a esa persona para informarle de tu comentario. Esto se llama una "@mención", porque estás mencionando a otra persona. También puedes `@mencionar` equipos dentro de una misma organización. Para más información, puedes consultar “[Recibir notificaciones acerca de la actividad en GitHub](https://help.github.com/categories/receiving-notifications-about-activity-on-github/) en *GitHub Help*.
```

Reply:
I left this one without the tilde because it won't render properly as a mention. Let me know if you still prefer to add it.

Reply:
It's ok for the @mencion.

On the article I disagree since there is an spanish article: https://docs.github.com/es/github/managing-subscriptions-and-notifications-on-github. (although we would have to add the /es part). The point is the page would me smart enough to pick the user preferred language